sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -12,7 +12,8 @@ export interface Context {
 }
 
 // @public
-export const context: ContextAPI;
+const context_2: ContextAPI;
```

Comment:
This is really bizarre to me. Is `context` also the name of a global... in this... context?